ProjectTemplates/ArcGISRuntime.AR/Installer/source.extension.vsixmanifest

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="Esri.ArcGISRuntime.ARTemplates" Version="1.2.0" Language="en-US" Publisher="Esri" />
+        <Identity Id="Esri.ArcGISRuntime.ARTemplates" Version="1.3.0" Language="en-US" Publisher="Esri" />
         <DisplayName>ArcGIS Runtime SDK for .NET - AR Templates</DisplayName>
         <Description xml:space="preserve">Project templates for building Augmented Reality apps for Android and iOS with Xamarin and ArcGIS Runtime SDK for .NET.</Description>
         <MoreInfo>https://links.esri.com/runtime-dotnet-sdk</MoreInfo>
         <License>EULA.rtf</License>
+        <ReleaseNotes>Updated to ArcGIS Runtime v100.11.0</ReleaseNotes>
```

Comment:
Is this permitted in the spec?
> Specifies a file (.txt, .rtf) that contains release notes. Also takes the URL of a website that displays the release notes.

Reply:
Odd that it let me enter a string. I'll remove